UI/lib/elements.html

```diff
@@ -424,6 +424,12 @@
 
 <?lsmb BLOCK CLOSE_STATUS_DIV ?></div><?lsmb END ?>
 
+<?lsmb BLOCK decorated_text;
+  IF element_data && element_data.msg.defined();
+    decorated_text(element_data.msg.replace('&lt;((/?[ibup]|br))&gt;','<$1>'));
```

Comment:
What helps the reader to understand that "decorated_text()" means that the text is being translated?

Also, you're translating here, but you already did that here: https://github.com/ledgersmb/LedgerSMB/blob/master/lib/LedgerSMB/Scripts/setup.pm#L776 ; so this function (decorated_text(), that is) is doing double work.

Given that all variable/dynamic values already have been translated when going into the template, I'd prefer the approach in https://github.com/ledgersmb/LedgerSMB/blob/master/lib/LedgerSMB/Scripts/setup.pm#L776 rather than the one introduced here.